src/header/header.scss

```diff
@@ -25,159 +25,105 @@
     color: t(iui-text-color);
   }
 
-  > .iui-left,
-  > .iui-center,
-  > .iui-right {
-    display: flex;
-    align-items: center;
-    height: 100%;
-  }
-
-  @include iui-header-buttons;
-
-  > .iui-left {
-    > nav {
-      display: flex;
-      align-items: center;
-      height: 100%;
-    }
-
-    &,
-    > nav {
-      overflow: hidden;
-      box-sizing: content-box;
-      margin-bottom: -$iui-xs;
-      padding-bottom: $iui-xs;
-    }
+  &.iui-slim {
+    height: round($iui-baseline * 2.5) + $iui-xxs; // Remove `+ $iui-xxs` when box-sizing: border-box is applied to all elements.
 
-    .iui-chevron {
-      @include iui-icons-small;
-      flex-shrink: 0;
-      margin: 0 $iui-xs;
+    .iui-user-icon {
```

Comment:
I guess ideally we could have added class on this one too... ? Not sure if that was better, though